spyder_kernels/console/kernel.py

```diff
@@ -361,6 +374,71 @@ def get_source(self, objtxt):
         if valid:
             return getsource(obj)
 
+    # -- For Matplolib
+    def get_matplotlib_backend(self):
+        """Get current matplotlib backend."""
+        try:
+            import matplotlib
+            return MPL_BACKENDS_TO_SPYDER[matplotlib.get_backend()]
+        except ImportError:
```

Comment:
```suggestion
        except Exception:
```

Better to be safe instead of generating an unnecessary error.